### ISSUE TITLE
Segmentation fault sometimes on Linux while starting desktop window mode

### ISSUE THREAD

**Issue by shalithasuranga** [neutralinojs/neutralinojs#118]

**Comment by Blatman** [comment]
MacOS 10.13.6 - High Sierra using v1.2.0 (latest download)

Occurs on MacOS as well (segmentation fault 11) - have attached a test app to demonstrate, app will crash on start.
Seems to depend on where the files are placed and what is regarded as the webroot. I found it to be a problem if the index.html in the app folder was referencing an image in the same folder but in a subfolder (ie: outside the assets folder). The problem ceases if the image path was changed from images/saddo.png to assets/images/saddo.png
To avoid the issue I ran the webroot within the assets folder but this would be silly normally.

[simpletest.zip](https://github.com/neutralinojs/neutralinojs/files/3990253/simpletest.zip)

BTW
Potentially not a bad project but no binary image support is a showstopper and I couldn't disable the debug terminal that opens with the app window. Also noticed that the neutralino binary would run ok on double click from the folder but when packaged as a .app I got a message saying it needs 10.14 or later to run.
Cheers.
